tests/config/test_config.yml

```diff
@@ -0,0 +1,143 @@
+---
+- name: Playbook to handle users
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: return current values of the global configuration options
+    ipaconfig:
+      ipaadmin_password: password01
```

Comment:
Please, change all testing passwords to `SomeADMINpassword`. It eases execution of all tests.